.github/workflows/ci.yml

```diff
@@ -35,6 +35,7 @@ jobs:
     - name: Install xi
       run: sudo apt-get install -y libxi-dev libglu1-mesa-dev
     - run: npm install
+    - run: npm run build
```

Comment:
this seems wrong

Reply:
@rom1504 the `package` script breaks ci, [see earlier](https://github.com/PrismarineJS/prismarine-viewer/pull/358#issuecomment-1438897862)